docs/_posts/2019-05-19-seminar-minutes-2020-09.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Bot Friday 2020 年第 9 次活动纪要
+author: likai
+categories: events
+tags:
+  - minutes
+header:
+  teaser:
```

Comment:
please add a teaser image, so that our blog post will be able to show with images in the topic list.